index.bs

```diff
@@ -239,48 +239,25 @@ The <dfn>rotation rate</dfn> measures the rate at which the device rotates about
 
 Note: [[MOTION-SENSORS]] and [[GYROSCOPE]] both contain a more detailed discussion of gyroscopes, rotation rates and measurements.
 
-Permissions Policy integration {#permissions-policy-integration}
+Permissions {#permissions-integration}
 ==============================
 
 <div class="issue">
 This integration is [=at risk=] due to a lack of test coverage in Web Platform Tests.
 </div>
 
-This specification defines the following <a>policy-controlled features</a>:
+This specification is classified as a [=powerful feature=] and, as such, it defines the following [=permissions=] and [=default allowlists=]:
```

Comment:
Let's be specific that this is the type of powerful feature which is a policy-controlled feature:

```suggestion
This specification is a [=powerful feature=] and, as such, it defines the following [=permissions=] which are [=policy-controlled features=] with the given [=default allowlists=]:
```